Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -294,12 +297,25 @@ public static class Loader {
                             }
                             continue;
                         }
+                        if (entry.FileName == ".everestignore") {
+                            List<string> lines = new();
+                            using (MemoryStream stream = entry.ExtractStream())
+                            using (var reader = new StreamReader(stream)) {
```

Comment:
```suggestion
                            using (StreamReader reader = new StreamReader(stream)) {
```
More code style consistency, especially here where there's two other similar paragraphs.